blocks/editable/index.js

```diff
@@ -298,7 +299,7 @@ export default class Editable extends Component {
 
 	isStartOfEditor() {
 		const range = this.editor.selection.getRng();
-		if ( range.startOffset !== 0 || ! range.collapsed ) {
+		if ( ! isAtCursorStart( range.startContainer, range.startOffset ) || ! range.collapsed ) {
```

Comment:
Could we replace these methods entirely?

Reply:
As is use `isEdge( editor.getBody(), true )` for isStart and `isEdge( editor.getBody(), false )` for end ? I think you're right. Nice catch :)

Reply:
Or alternatively exposing isLeftEdge and isRightEdge?

Reply:
I'll defer to your preference :) I guess it matches the name in the range API for collapsing, which is most likely what you're basing it on. I've always found start a very strange name for a boolean, but that's just person taste.

I'm working on this now, so are you OK with the idea of exposing constants for the second argument or would you prefer passing an object like your last suggestion?

Reply:
I think an object might be better for now, since it's just adding the braces. :) But that's just my opinion.

Reply:
Used object notation as recommended.